subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/matcher/TextMatchers.kt

```diff
@@ -0,0 +1,136 @@
+package com.avito.android.test.matcher
+
+import android.content.res.Resources
+import android.view.View
+import android.widget.TextView
+import androidx.annotation.StringRes
+import androidx.test.espresso.matcher.BoundedDiagnosingMatcher
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.Matchers
+import org.hamcrest.StringDescription
+
+internal object TextMatchers {
```

Comment:
Let's add comments about original classes as a source.
It may help for further maintainers.

Reply:
Put origination comments on matcher classes.